src/SUMMARY.md

```diff
@@ -29,4 +25,6 @@
 		- [Pagging](x86_64/premiers-pas/05-pagging.md)
 		- [Multitâche](x86_64/premiers-pas/06-multitache.md)
 		- [Tache utilisateur](x86_64/premiers-pas/06-tache-utilisateur.md)
-		- [Epilogue](x86_64/premiers-pas/06-epilogue.md)
\ No newline at end of file
+		- [Epilogue](x86_64/premiers-pas/06-epilogue.md)
+- [générique]()
```

Comment:
J'aime pas l'idée d'une section "générique" car ca ne veux rien dire, pourquoi ne pas la nommé 'Gestion de la mémoire'

Reply:
si on a une section pour chaque arch faire une section générique avec en sous dossier gestion de la mémoire fait sens

Reply:
C'est un niveau d'indirection de trop IMO

Reply:
non pour le coup ça fait sens dans la hierarchie des dossiers, 

genre pour prendre l'exemple de mon kernel j'ai:
generic
i686
aarch64
arm
x86_64

et   `generic` a pour but de rassembler tout ce qui se place légèrement au dessus des implémentation spécifique aux archs

Reply:
Si on veut structurer la doc comme un livre c'est mieux de faire:

1. Introduction

2. Gestion de la mémoire

3. Gestion des processus

4. etc

5. Implementation
    5.1 x86
    5.2 arm
    5.3 etc
    
Ca permetrais au gens de prendre les base avant de ce lancer dans le developement ;)

Reply:
Je suis pas d'accord c'est très dépendant de l'implémentation, faudrait au contraire faire des dossier pour chaque implémentation et au dessus faire des dossier purements théoriques (qui a mon sens iraient dans générique (ou autre nom))

Reply:
Pas vraiment la théorie des system d'exploitation de ne dépend pas l'implémentation

Reply:
y'a beaucoup de concepte qui ne sont pas universel ou complement diff dans des plateforme autre que x86

Reply:
Je vois ces chapitres comment étant théorique, genre expliquer les diffèrent algorithme pour l'allocation de mémoire etc sans rentre dans des détails d'implémentations

Reply:
faut des chapitre théorique, mais le problème c'est qu'ils ne peuvent pas être les memes en fonction de l'arch (par ex pas mal de concepte inéxistant d'une arch a l'autre comme par ex la GDT)

Reply:
D'ou ma défense d'un dossier générique (ou ce que vous voulez) qui rassemble ce qui est commun

Reply:
L'exemple le plus simple serait par exemple la RS-232 la théorie est commune, mais l'implémentation entre x86, une rpi (voir entre les RPI avec mini-uart) etc est completement différente

Reply:
Bah justement le but c'est d'introduire les concept de haut niveau, sans parler de GDT ou de RS-232